src/main/resources/templates/subscription.html

```diff
@@ -262,13 +262,21 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                         <!--  AUTHENTICATION  -->
                         <div class="pt-3 form-group">
                             <label class="h5 pl-1 control-label font-weight-bold">Authorization</label>
+                            <img class="cursor-pointer" id="authInfo" alt="Auth Info"
+                                src="assets/images/information.png" data-toggle="tooltip" data-placement="top"
+                                title="This selector enables you to select differen authentication methods. <br>
+                                    NO_AUTH: No Authorization used<br>
+                                    BASIC_AUTH: Username and password will be Base 64 encoded<br>
+                                    BASIC_AUTH Jenkins CSRF Protection (crumb): Username and password will be Base 64 encoded.
+                                    A crumb will be fetched automatically before request is made. (Currently default in many Jenkins instances)
+                                    Will work even when CSRF is disabled in Jenkins" />
                             <div>
                                 <select id="selectAuth" title="Choose an authentication type" data-bind="options: $root.authenticationType_in,
                                     optionsText: 'text', optionsValue: 'value', value: authenticationType">
                                 </select>
                             </div>
                         </div>
-                        <div class="p-1 border form-group" data-bind="visible: authenticationType() == 'BASIC_AUTH'">
+                        <div class="p-1 border form-group" data-bind="visible: authenticationType() != 'NO_AUTH'">
```

Comment:
This is valid at the moment, otherwise i think it would be okay to say =="BASIC_AUTH" or == "BASIC_AUTH_JENK......." since maybe only those two will require username/password.

Reply:
This part may have to be updated for every new authentication type, this looks cleanest with the ones we have right now.